bears/c_languages/codeclone_detection/ClangComplexityBear.py

```diff
@@ -0,0 +1,98 @@
+from clang.cindex import Index, CursorKind
+
+from coalib.bears.GlobalBear import GlobalBear
+from bears.c_languages.ClangBear import clang_available
+from coalib.results.Result import Result
+
+
+class ClangComplexityBear(GlobalBear):
```

Comment:
why not make it local? You read the files one by one anyway, right? Let coala parallelize.